components/automate-ui/src/app/pages/project/details/project-details.component.ts

```diff
@@ -69,13 +69,15 @@ export class ProjectDetailsComponent implements OnInit, OnDestroy {
 
     combineLatest([
       this.store.select(getStatus),
-      this.store.select(updateStatus)
+      this.store.select(updateStatus),
+      this.store.select(getAllStatus)
```

Comment:
this is why we were showing stale rules: we weren't waiting for the fresh rules request to finish loading before rendering the table